pages/words/[id].js

```diff
@@ -37,57 +37,55 @@ export default function SingleWordPage() {
-          <>
-            <h1>Word entry </h1>
-            <GoBackButton onClick={() => router.push("/words")} />
-            <EditButton onClick={() => setIsShowMode(false)}></EditButton>
-            {!entry ? null : (
-              <SingleEntry
-                isFavorite={entry.isFavorite}
-                word={entry.word}
-                language={entry.language}
-                translated={entry.translated}
-                notes={entry.notes}
-                onDeleteEntry={() => {
-                  router.push("/words");
-                  setTranslationList(
-                    translationList.filter(
-                      (translation) => translation.id !== id
-                    )
-                  );
-                }}
-                onToggleFavorite={() =>
-                  setTranslationList(
-                    translationList.map((translation) =>
-                      translation.id === id
-                        ? {
-                            ...translation,
-                            isFavorite: !translation.isFavorite,
-                          }
-                        : translation
-                    )
-                  )
-                }
-              ></SingleEntry>
-            )}
-          </>
+          <EditButton onClick={() => setIsShowMode(false)}></EditButton>
         ) : (
-          <>
-            <h1>Edit word</h1>
-            <CancelDeleteButton onClick={() => setIsShowMode(true)} />
-            {!entry ? null : (
-              <Form
-                entry={entry}
-                isEditMode={true}
-                onSubmitEvent={handleEditEntry}
-              />
-            )}
-          </>
+          <CancelEditButton onClick={() => setIsShowMode(true)} />
         )}
-      </main>
-    </>
+        {!entry ? null : (
+          <SingleEntry
+            isFavorite={entry.isFavorite}
+            word={entry.word}
+            language={entry.language}
+            translated={entry.translated}
+            notes={entry.notes}
+            onDeleteEntry={() => {
+              router.push("/words");
+              setTranslationList(
+                translationList.filter((translation) => translation.id !== id)
+              );
+            }}
+            onToggleFavorite={() =>
+              setTranslationList(
+                translationList.map((translation) =>
+                  translation.id === id
+                    ? {
+                        ...translation,
+                        isFavorite: !translation.isFavorite,
+                      }
+                    : translation
+                )
+              )
+            }
+          ></SingleEntry>
+        )}
+      </>
+      {!isShowMode && (
+        <>
+          {!entry ? null : (
+            <Form
+              type="edit"
+              entry={entry}
+              isEditMode={true}
+              onSubmitEvent={handleEditEntry}
+            />
+          )}
+        </>
```

Comment:
You need no fragments here

Reply:
No, I need them, it's not working without!